src/error.rs

```diff
@@ -325,18 +325,21 @@ impl<I: fmt::Display + Hash + Eq, S: Span> fmt::Display for Simple<I, S> {
                     None => format!("end of input"),
```

Comment:
and I think you missed a change in the few lines above this line (but I can't put a comment there)

Reply:
Got it.

I really appreciate your time and your help. Thank you for reviewing this with me. I will make the changes that you suggest.